google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreImpl.java

```diff
@@ -165,42 +204,30 @@ public T call() throws DatastoreException {
 
   @Override
   public <T> T runInTransaction(final TransactionCallable<T> callable) {
-    com.google.cloud.datastore.telemetry.TraceUtil.Span span =
```

Comment:
What is the reason that we remove startSpan here and at L221 (edited version) below?

Reply:
So for now, I've replaced the `TraceUtil.startSpan` with `io.opentelemetry.api.trace.SpanBuilder` - `TraceUtil.startSpan` is really a wrapper around `io.opentelemetry.api.trace.SpanBuilder`.
I need to debug why using `TraceUtil.startSpan` doesn't work as expected (it has probably something to do with how the parent context is stored and propagated via thread-local storage but I'm not sure). 

I don't think this issue is a blocker and I can debug this in the background as we make progress with rest of the tracing work.